package.json

```diff
@@ -39,7 +39,7 @@
     "test.utils": "nx test utils"
   },
   "simple-git-hooks": {
-    "pre-commit": "pnpm format.staged"
+    "pre-commit": "pnpm format.staged && pnpm lint"
```

Comment:
Added this so that linting errors are caught before creating a PR. Let me know if this was intentionally excluded.